interactive.js

```diff
@@ -131,6 +162,48 @@ const handleFkillError = async processes => {
 	}
 };
 
+const DEFAULT_EXIT_TIMEOUT = 3000;
+
+const performKillSequence = async processes => {
+	if (!Array.isArray(processes)) {
+		processes = [processes];
+	}
+
+	let survived;
+	let hadError;
+	try {
+		await fkill(processes);
+		const exited = await Promise.all(processes.map(process => processExited(process, DEFAULT_EXIT_TIMEOUT)));
+		survived = processes.filter((_, i) => !exited[i]);
+	} catch (error) {
+		survived = processes;
+		hadError = error;
+	}
+
+	if (survived.length > 0) {
+		const suffix = survived.length > 1 ? 'es' : '';
+		const problemText = hadError ? `Error killing process${suffix}.` : `Process${suffix} didn't exit in ${DEFAULT_EXIT_TIMEOUT}ms.`;
+
+		if (process.stdout.isTTY === false) {
```

Comment:
```suggestion
		if (!process.stdout.isTTY) {
```

and early-return here too.

Reply:
Are you sure about isTTY part? It was there since interactive mode introduction and it feels like it's that specific for some reason.

Reply:
It's documented to be a boolean.

Reply:
Yeah, but even in docs they do it like
```
$ node -p -e "Boolean(process.stdout.isTTY)"
true
$ node -p -e "Boolean(process.stdout.isTTY)" | cat
false
```

Reply:
Anyway, ok, let's try and see.